versions/latest/modules/en/pages/install/quickstart.adoc

```diff
@@ -33,13 +38,22 @@ This will install the `rke2-server` service and the `rke2` binary onto your mach
 systemctl enable rke2-server.service
 ----
 
+.  To pull images from the Rancher Prime registry, set the following value in `etc/rancher/rke2/config.yaml`:
```

Comment:
Is the method mentioned in this [comment](https://github.com/rancher/rke2-product-docs/issues/49#issuecomment-3228717059) replaced by what's listed here?